kubetest2-gke/deployer/network_test.go

```diff
@@ -32,24 +32,6 @@ func TestPrivateClusterArgs(t *testing.T) {
 		clusterInfo    cluster
 		expected       []string
 	}{
-		{
-			desc:           "no private cluster args are needed for non-private clusters",
-			projects:       []string{"project1"},
-			network:        "whatever-network",
-			accessLevel:    "",
-			masterIPRanges: []string{"whatever-master-IP-range"},
-			clusterInfo:    cluster{index: 0, name: "whatever-cluster-name"},
-			expected:       []string{},
-		},
-		{
-			desc:           "--private-cluster-master-ip-range can be empty for non-private clusters",
-			projects:       []string{"project1"},
-			network:        "whatever-network",
-			accessLevel:    "",
-			masterIPRanges: []string{},
-			clusterInfo:    cluster{index: 0, name: "whatever-cluster-name"},
-			expected:       []string{},
-		},
```

Comment:
why are we removing some tests?

Reply:
It's because I removed the `if accessLevel == ""` check in this function, so these tests become invalid.